src/notty.ml

```diff
@@ -209,10 +209,12 @@ module A = struct
   and b x = x land 0xff
 
   let bold      = 1
-  and italic    = 2
-  and underline = 4
-  and blink     = 8
-  and reverse   = 16
+  and dim       = 2
```

Comment:
Why have both faint and dim?

Reply:
Both words have been actively used to name this parameter, and I don't see a reason not to have aliases here, so I guess, why not? But of course the author @pqwy has the final say. Usually this kind of aliasing can be clarified in documentation, but the current module does not come with documentation.

Reply:
My personal preference would be to reduce redundancy but that's not my call here.

Reply:
That's a very sensible choice. My priority is to make the interface user-friendly (without sacrificing much efficiency).